deploy/stacks/container.py

```diff
@@ -174,12 +175,13 @@ def __init__(
             update_bucket_policies_task.task.security_groups
         )
 
```

Comment:
This is a #TODO but I think it is not that complicated to add parameters or read parameters. My concern is, how do we design something that scales? Are we going to have deployment changes/architectural components that change or are added based on modules selected?

Reply:
Yes, it shouldn't be difficult to implement, but I'd rather implement it in a dedicated PR after migrating the other dataset/sharing tasks 

I think that we should not replicate changes between config.json and cdk.json, otherwise it'd be mess of configuration. So, I suggest placing all information about modules (even if it can impact deployment) to config.json and consulting with it during the deployment. 
Of course, this approach has its own drawbacks, but it should be easy to implement and, more importantly, change if we decide to go this the other approach.
What do you think?